index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Portfolio - ZC</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="./icons/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <!-- Toolbar -->
+    <div class="toolbar">
+        <p id="toolbartext">Welcome</p>
+        <img id="menuicon" src="./icons/hamburguermenu.png" atl="Menu" />
+    </div>
+
+    <!-- Headline Section-->
+    <div class="headline">
+
+        <div class="container">
+            <h1>Hey there. I'm Cristian<br><span style="color: #36B37E"> I'm a software developer</span> </h1>
+        </div>
+
+        <div class="container">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+        </div>
+
+        <div class="container">
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/twitter.svg" alt="Twitter">
+            </div>
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/linkedinicon.svg" alt="LinkedIn">
+            </div>
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/medium.svg" alt="Medium">
+            </div>
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/github.svg" alt="GitHub">
+            </div>
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/angellist.svg" alt="Angel List">
+            </div>
```

Comment:
- [ ] You did well 😊 at adding the icons here, but kindly try to decrease the margin between the icons here to be like the Figma Design and with that, it will be nice for the visitors who will see your portfolio and communicate with you 😃

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/219167911-ae3160f0-2bc0-4cfe-a77c-ea0f97d0d4c6.png)    |     ![image](https://user-images.githubusercontent.com/83360636/219167791-ffb8654b-c75b-4187-97fe-7886d53688bf.png)  |